lib/master.js

```diff
@@ -23,7 +23,9 @@ function liveWorkerIds() {
       worker.process.exitCode == null;
     return isAlive;
   });
-  return workerIds;
+  return workerIds.sort(function(a, b) {
```

Comment:
Go home, JavaScript, you're drunk...

Fortunately, lodash's sort does the right thing:
```js
return _.sortBy(workerIds);
```

I'm not sure which is better, though... using Array.prototype.sort requires a numeric comparator, which one could naively assume isn't necessary, resulting in the reader assuming this is really a `.reverse()` done the "hard" way.

Using `lodash.sortyBy` without specifying a field to sort by raises the question of why `Array.prototype.sort` isn't being used..